engine/api/repositoriesmanager/repositories_manager.go

```diff
@@ -543,6 +606,10 @@ func (c *vcsClient) GrantWritePermission(ctx context.Context, repo string) error
 	return nil
 }
 
+func (c *vcsClient) GetAccessToken(_ context.Context) string {
+	return ""
```

Comment:
Is this useless ?

Reply:
need this to implement `VCSAuthorizedClient`interface